.github/workflows/support.yml

```diff
@@ -12,6 +12,7 @@ on:
       - "**/*.yml"
       - "tests/**"
       - "!tests/**/README.*"
+      - "!README_tests.md"
```

Comment:
Might as well leave these two changes out actually. These ```!``` s are negation symbols, so this entry has no effect. The entry means ```(all *.yml files AND everything under the tests directory) EXCEPT README's```; excluding ```README_tests.md``` has no effect, because its not part of the included set.

Reply:
OK; removed.